whylog/tests/tests_client/test_backtrackerSearcher.py

```diff
@@ -0,0 +1,62 @@
+from unittest import TestCase
+import os.path
+from whylog.tests.tests_client.constants import test_paths, AFewLinesLogParams
+
+from whylog.client import searchers
+
+
+class TestBacktrackSearcher(TestCase):
+    def _get_log_file_path(self):
+        prefix_path = os.path.join(*test_paths.path_test_files)
```

Comment:
`tests_client` could have a `test_config_path(filename)`. Then this line could not be copied in practically every test, before creating a variable containing path